examples/cdk/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "cdk-app",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "bin": {
     "cdk-app": "bin/cdk-app.js"
   },
   "scripts": {
     "build": "tsc --skipLibCheck",
     "watch": "tsc -w",
-    "test": "jest",
+    "test": "npm run build && jest",
     "cdk": "cdk"
   },
   "devDependencies": {
```

Comment:
Let's bump `@aws-lambda-powertools/commons` & others to latest version `^0.3.1`

Reply:
done in https://github.com/awslabs/aws-lambda-powertools-typescript/pull/446/commits/2852495933b01ababd5092c057e2a548711b7821